futurenhs.content.api/Umbraco9ContentApi.Test/Services/FutureNhsContentServiceTests.cs

```diff
@@ -64,7 +67,7 @@ public void Setup()
         public void GetFutureNhsContentService()
         {
             _futureNhsContentService = new(_mockPublishedContentQuery.Object,
-_mockFutureNhsContentResolver.Object, _mockContentService.Object);
+_mockFutureNhsContentResolver.Object, _mockContentService.Object, _mockLogger.Object);
```

Comment:
The indentation looks a little funky here, is this correct?

Reply:
It does, corrected indentation